Sources/NIOPerformanceTester/main.swift

```diff
@@ -770,3 +771,8 @@ try measureAndPrint(desc: "byte_buffer_view_iterator_1mb", benchmark: ByteBuffer
 
 try measureAndPrint(desc: "byte_to_message_decoder_decode_many_small",
                     benchmark: ByteToMessageDecoderDecodeManySmallsBenchmark(iterations: 1_000, bufferSize: 16384))
+
+measureAndPrint(desc: "generate_10k_random_request_keys") {
+    let keys = (0 ..< 10_000).map { _ in NIOWebSocketClientUpgrader.randomRequestKey() }
```

Comment:
Oh, actually, I can: instead of using `map` let's use something else that introduces less allocator overhead. Perhaps `reduce(into: 0, { $0 &+= NIOWebSocketClientUpgrader.randomRequestKey().count })`. That should prevent the optimiser from killing the benchmark, but won't force the allocator to do extra work.

Reply:
Very good point!
Will change in a few minutes